magic-the-gatheringTests/Presentation/Scenes/Category Scene/CategoryViewSpec.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CategoryViewSpec.swift
+//  magic-the-gatheringTests
+//
+//  Created by isabel.isaura.l.lima on 28/01/2019.
+//  Copyright © 2019 leonel.menezes.lima. All rights reserved.
+//
+
+import Foundation
+import Quick
+import Nimble
+
+@testable import magic_the_gathering
+
+class CategoryViewSpec: QuickSpec {
+    
+    override func spec() {
+        
+        let cardMock = Card.mock()
```

Comment:
Se for usar o `describe` mover isso para dentro do bloco do `describe` isso faz parte do teste.

Reply:
Done!